service/storage_service.cc

```diff
@@ -715,6 +715,7 @@ future<> storage_service::topology_state_load() {
     for (const auto& gen_id : _topology_state_machine._topology.committed_cdc_generations) {
```

Comment:
```
The newly created generation is seen in system_distributed.cdc_generation_timestamps
but not in system.cdc_generations_v3, thus there are used streams that the CDC
based on raft doesn't know about.

Save the last CDC generation in the system.cdc_local during loading the topology
state so that it is visible for CDC not based on raft.
```

Please correct me if I'm wrong:  _The newly created generation_ is created after restarting the node in the recovery mode because it doesn't see the latest generation in `system.cdc_local`. Instead of creating the new generation, the node should see and write to the generation already committed to group0.

Reply:
Yes, that's correct.